packages/paste-core/components/alert/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import {render} from 'react-dom';
+import {ReactWrapper, mount} from 'enzyme';
+import {axe} from 'jest-axe';
+import {Theme} from '@twilio-paste/theme';
+import {Alert} from '../src';
+
+const onDismissMock: jest.Mock = jest.fn();
+
+describe('Alert', () => {
+  describe('Dismiss button', () => {
+    it('Should add a dismiss button when onDismiss is passed as a function to call', () => {
+      const eventHandlerMock: jest.Mock = jest.fn();
+      const wrapper: ReactWrapper = mount(
+        <Alert onDismiss={eventHandlerMock} variant="neutral">
+          This is an alert
+        </Alert>
+      );
+      expect(wrapper.find('button').length).toEqual(1);
+    });
+
+    it('Should call the onDismiss event handler when close button clicked', () => {
+      const eventHandlerMock: jest.Mock = jest.fn();
+      const wrapper: ReactWrapper = mount(
+        <Alert onDismiss={eventHandlerMock} variant="neutral">
+          This is an alert
+        </Alert>
+      );
+      wrapper.find('button').simulate('click');
+      expect(eventHandlerMock).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('Aria roles', () => {
+    it('Should add the role of status to the neutral alert', () => {
+      const wrapper: ReactWrapper = mount(<Alert variant="neutral">This is an alert</Alert>);
+      expect(wrapper.exists('[role="status"]')).toEqual(true);
+    });
+
+    it('Should add the role of alert to the error alert', () => {
+      const wrapper: ReactWrapper = mount(<Alert variant="error">This is an alert</Alert>);
+      expect(wrapper.exists('[role="alert"]')).toEqual(true);
+    });
+
+    it('Should add the role of alert to the warning alert', () => {
+      const wrapper: ReactWrapper = mount(<Alert variant="warning">This is an alert</Alert>);
+      expect(wrapper.exists('[role="alert"]')).toEqual(true);
+    });
+
+    it('Should add the provided role of alert to the alert', () => {
```

Comment:
`'Should add the provided role of alert to the alert` -> `'Should add the provided role to the alert`?